contracts/token/ERC20/extensions/ERC20FlashMint.sol

```diff
@@ -14,6 +14,10 @@ import {ERC20} from "../ERC20.sol";
  * Adds the {flashLoan} method, which provides flash loan support at the token
  * level. By default there is no fee, but this can be changed by overriding {flashFee}.
  *
+ * NOTE: When this extension is used along with the {ERC20Capped} extension, its cap
+ * won't be checked in {flashLoan}. Although it can be limited by also overriding {maxFlashLoan}
+ * to also cap the maximum amount of tokens that can be flash-minted.
+ *
```

Comment:
I'm not sure about the formulation here. 

In `flashLoan`, the mint operation has to follow the rules of `_update`, which is what `ERC20Capped` overrides. So the cap fill be respected when minting the "flash" tokens. It is correct to say that `maxFlashLoan` doesn't reflect that by default, but its IMO incorrect to say that "its cap won't be checked in {flashLoan}"

Reply:
I went with @frangio's [suggestion](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4398#discussion_r1247963082), I think this is clearer and removes the "its cap won't be checked in {flashLoan}"